lib/streamlit/hashing.py

```diff
@@ -493,6 +494,15 @@ def _to_bytes(self, obj, context):
         elif type_util.is_type(obj, "tensorflow.python.client.session.Session"):
             return self.to_bytes(id(obj))
 
+        elif type_util.is_type(obj, "builtins.PyCapsule"):
+            ctypes.pythonapi.PyCapsule_GetPointer.restype = ctypes.c_void_p
```

Comment:
Add a comment explaining what you're hashing here. I have no idea what's going on!